internal/factsengine/gatherers/cibadmin.go

```diff
@@ -0,0 +1,73 @@
+package gatherers
+
+import (
+	log "github.com/sirupsen/logrus"
+	"github.com/trento-project/agent/pkg/factsengine/entities"
+	"github.com/trento-project/agent/pkg/utils"
+)
+
+const (
+	CibAdminGathererName = "cibadmin"
+)
+
+// nolint:gochecknoglobals
+var (
+	CibAdminCommandError = entities.FactGatheringError{
+		Type:    "cibadmin-command-error",
+		Message: "error running cibadmin command",
+	}
+
+	CibAdminDecodingError = entities.FactGatheringError{
+		Type:    "cibadmin-decoding-error",
+		Message: "error decoding cibadmin output",
+	}
+)
```

Comment:
@CDimonaco are you aware of some other way to declare this stuff both cleanly _and_ without pissing off the linter? I don't like globals, but I have to say I like what @arbulu89 did here.

Reply:
In any case, is not something I made up here. We have been using this way in all the gatherers since we started the porting